### ISSUE TITLE
Maker: the MaterialEditor window is broken in many ways

### ISSUE THREAD

**Issue by mosirnik** [mosirnik/KK_MainGameVR#39]
An incomplete list of problems:

* Clicking on the titlebar causes the window to disappear.
* Clicking on a dropdown list doesn't do anything.

**Comment by mosirnik** [comment]
I did some investigation.

* Dragging the window is broken because the code assumes that screen space equals world space at https://github.com/IllusionMods/KK_Plugins/blob/13e77cfc3f6f7d4b3027c9b9cf092bfeac365cbb/src/UIUtility/MovableWindow.cs#L31.
* I still don't know why dropdown lists don't work.
* Some buttons open a new operating-system window and defocuses the Koikatsu window. This leads to various issues.

Other UI items seem to work fine. This includes buttons, sliders and text inputs.

**Comment by TheWendigoAlpha** [comment]
yeah i get the same issue aswell none of the dropdown menus work at all or dont even show yet i can still select the options if i press where they are supposed to be

**Comment by mosirnik** [comment]
@TheWendigoAlpha That's a very useful observation, thanks!
